package.json

```diff
@@ -19,22 +19,28 @@
   "bin": "bin/peggy",
   "scripts": {
     "clean": "rimraf -f build && rimraf -f browser && mkdir browser",
-    "terser": "terser -c passes=2 -m --sequences 40 build/rollup/peggy.umd.js > browser/peggy.min.js && terser -c passes=2 -m --sequences 40 --module build/rollup/test.umd.js > docs/js/test-bundle.min.js && terser -c passes=2 -m --sequences 40 --module build/rollup/benchmark.umd.js > docs/js/benchmark-bundle.min.js",
-    "rollup": "cp package.json build/ts && rollup -c",
-    "ts": "tsc --build tsconfig.json",
+    "parser": "node bin/peggy -o lib/parser.js --format commonjs src/parser.pegjs",
     "set_version": "node ./tools/set_version",
-    "headers": "node ./tools/header.js browser/peggy.min.js docs/js/benchmark-bundle.min.js docs/js/test-bundle.min.js",
-    "test": "jest --coverage --testPathIgnorePatterns=build",
     "lint": "eslint . --ext .js bin/peggy",
+    "ts": "tsc --build tsconfig.json",
+    "test": "jest --coverage --testPathIgnorePatterns=build",
     "benchmark": "node ./benchmark/run_bench.js",
-    "parser": "node bin/peggy -o lib/parser.js --format commonjs src/parser.pegjs",
-    "deploy": "cp browser/peggy.min.js docs/vendor/peggy/peggy.min.js",
-    "audit": "npm run test && npm run benchmark",
-    "make": "npm run parser && npm run ts && npm run rollup && npm run terser ",
-    "setup": "npm run clean && npm run set_version",
+    "rollup": "cp package.json build/ts && rollup -c",
+    "terser": "npm run terser:peggy && npm run terser:tests && npm run terser:bench",
+    "terser:peggy": "terser -c passes=2 -m --sequences 40 build/rollup/peggy.umd.js -o build/peggy.min.js",
+    "terser:tests": "terser -c passes=2 -m --sequences 40 --module build/rollup/test.umd.js -o build/test-bundle.min.js",
+    "terser:bench": "terser -c passes=2 -m --sequences 40 --module build/rollup/benchmark.umd.js -o build/benchmark-bundle.min.js",
+    "headers": "node ./tools/header.js build/peggy.min.js build/benchmark-bundle.min.js build/test-bundle.min.js",
+    "deploy": "npm run deploy:peggy && npm run deploy:tests && npm run deploy:bench",
+    "deploy:peggy": "cp build/peggy.min.js docs/vendor/peggy/peggy.min.js && cp build/peggy.min.js browser/peggy.min.js",
+    "deploy:tests": "cp build/test-bundle.min.js docs/js/test-bundle.min.js",
+    "deploy:bench": "cp build/benchmark-bundle.min.js docs/js/benchmark-bundle.min.js",
+    "build": "npm run build-notidy && npm run tidy",
     "build-notidy": "node ./tools/announce \"Static source analysis\" && npm run lint && node ./tools/announce \"Setting up the build\" && npm run setup && node ./tools/announce \"Making libraries\" && npm run make && node ./tools/announce \"Running Audits\" && npm run audit && node ./tools/announce \"Creating distributions\" && npm run headers && npm run deploy && node ./tools/announce \"Complete!\"",
```

Comment:
remove announce in this patch, please.

Reply:
You mean, remove just calling `node ./tools/announce` or `build`, `build-notidy` and `tidy` completely?

Reply:
remove the script `tools/announce` and all of its use in `package.json`.  Particularly now that the steps are more easy to see in GHA, there's no need for it.

Reply:
Script removed